src/js/integrations/sdkinfo.ts

```diff
@@ -64,3 +64,12 @@ export class SdkInfo implements Integration {
     });
   }
 }
+
+//this is 4.0 feature and cant be downleveled
+type Strings = [string, string];
+type Numbers = [number, number];
+export type StrStrNumNumBool = [...Strings, ...Numbers, boolean];
+
+//this will be downleveled
+type World = "world";
+export type Greeting = `hello ${World}`;
```

Comment:
This is an example of code that will be downleveled and code that can not be downleveled. When CI is done, I will link its output.